config/config.yaml

```diff
@@ -40,15 +40,15 @@ files:
 # Filter settings
 filter:
   # Require nearly full-length genomes.
-  min_length: 25000
+  min_length: 27000
 
   # Omit sequences with incomplete date annotations.
   exclude_where: "date='2020' date='2020-01-XX' date='2020-02-XX' date='2020-03-XX' date='2020-04-XX' date='2020-05-XX' date='2020-06-XX' date='2020-01' date='2020-02' date='2020-03' date='2020-04' date='2020-05' date='2020-06'"
 
   # Select a subset of sequences per unique geographic and temporal group.
   # This limits the number of sequences in the base build to prevent long runtimes.
   group_by: "division year month"
-  sequences_per_group: 2000
+  sequences_per_group: 200000
```

Comment:
When setting the value this high, does this effectively disable the grouping part of the filter rule? Can we make the `group_by` and `sequences_per_group` config entries optional and then delete them here?

Reply:
deleted those as well